src/ess/powder/correction.py

```diff
@@ -424,19 +339,17 @@ def _shallow_copy(da: sc.DataArray) -> sc.DataArray:
 
 
 def subtract_background(
-    data: FocussedDataDspacing[SampleRun],
+    data: IofDspacing,
     background: FocussedDataDspacing[BackgroundRun],
```

Comment:
Shoudn't this be `IofDspacing[BackgroundRun]`? In other words, don' we need to parametrize `IofDspacing`?

Reply:
Done. But this required changing `IofDspacing` to a generic type. So this is a breaking change to the workflow which needs to request `IofDspacing[SampleRun]` now. I had coded `SampleRun` in some places to avoid errors from using the same type twice as an argument. E.g.,
```python
def subtract_background(
    data: IofDspacing[RunType],
    background: IofDspacing[BackgroundRun],
) -> EmptyCanSubtractedIofDspacing[RunType]:
    ...
```
Would result in an instantiation that uses `IofDspacing[BackgroundRun]` for both arguments and Sciline doesn't like that.

Reply:
Given that tests passed with this "bug" (before your latest changes): Is there a test missing that actually checked the result of the background subtraction?

Reply:
We don't have any regression tests for the final result. But they would have been broken in this case anyway.
I don't know how we can test that the background subtraction is applied correctly. Unfortunately, the data are dimensionless at this point. Do you have an idea?

Reply:
Check that the workflow gives `(A - B)/C`, locally?

Reply:
See new commit